crates/bevy_winit/src/system.rs

```diff
@@ -296,6 +301,12 @@ pub(crate) fn changed_window(
                 ));
             }
 
+            if window.window_theme() != cache.window.window_theme() {
+                if let Some(theme) = window.window_theme() {
+                    winit_window.set_theme(Some(convert_window_theme(theme)));
+                }
```

Comment:
If the user wants to unset the value, it will not do it. It should pass a `None` when the theme is `None`. It currently doesn't.

```suggestion
                let theme = window.window_theme();
                winit_window.set_theme(theme.map(Into::into)));
```